pkg/porter/explain.go

```diff
@@ -300,6 +302,26 @@ func generatePrintable(bun cnab.ExtendedBundle, action string) (*PrintableBundle
 	}
 	sort.Strings(pb.Mixins)
 
+	// copy custom data to output, but ignore the Porter built-in data
+	ignoreKeyPrefixes := []string{
+		"io.cnab",
+		"sh.porter",
+	}
+
+	for key, value := range bun.Custom {
+		var ignore bool = false
+
+		for _, ignoreKeyPrefix := range ignoreKeyPrefixes {
+			if strings.HasPrefix(key, ignoreKeyPrefix) {
+				ignore = true
```

Comment:
nit: Add a break here

Reply:
The newly created function returns immediately if the condition is true so no break required anymore.